hatch.toml

```
@@ -44,6 +44,8 @@ fmt = [
     "black {args:.}",
     "ruff --fix {args:.}",
     "style",
+    "typing",
+    "spelling",
```

Should we include these scripts as well?
```suggestion
    "spelling",
    "riot",
    "security",
    "test",
    "suitespec-check",
```